docs/core/tools/dotnet-store.md

```diff
@@ -0,0 +1,87 @@
+---
+title: dotnet store command (.NET Core SDK 2.0 Preview 2) | Microsoft Docs
+description: The 'dotnet store' command stores the specified assemblies in the runtime package store.
+keywords: dotnet store, dotnet store, CLI, CLI command, .NET Core
+author: bleroy
+ms.author: mairaw
+ms.date: 06/26/2017
+ms.topic: article
+ms.prod: .net-core
+ms.technology: dotnet-cli
+ms.devlang: dotnet
+ms.assetid: 1e8e4122-8110-4b48-afce-afffb6737776
+---
+
+# dotnet store (.NET Core SDK 2.0 Preview 2)
```

Comment:
Remove it from the title and H1 .... but leave the includes warning? That's what I'll do, but let me know if you mean *everywhere*.